Assets/BCI/ControllerScripts/Controller.cs

```diff
@@ -654,13 +657,59 @@ public IEnumerator ReceiveMarkers()
                     //print("WE GOT A RESPONSE");
                     print("response : " + responseString);
 
+                    // If there are square brackets then remove them
+                    responseString = String.Concat(responseString.Split('['));
```

Comment:
If the desired outcome is to remove all instances of `[`  , `]` and `.` you could use string.replace.
e.g. `responseString.Replace("[", "").Replace("]","").Replace(".", "");`
This would save you having to split and concat multiple times.